ui/components/component-library/avatar-badge/avatar-badge.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box/box';
+
+export const badgePositions = ['top', 'bottom'];
+
+export const AvatarWithBadge = ({
+  children,
+  badgePosition,
+  badge,
+  ...props
+}) => {
+  return (
+    <Box className="avatar-badge" {...props}>
+      {/* Jazzicon , Avatar Account */}
+      {children}
+
+      <Box
+        className={
+          badgePosition === 'top'
+            ? 'avatar-badge-token-position-top'
+            : 'avatar-badge-token-position-bottom'
+        }
+      >
+        {badge}
```

Comment:
Could add some docs here too
```suggestion
        {/* Generally the AvatarNetwork at SIZES.XS */}
        {badge}
```

Reply:
Updated